Run `ruff check`

```
Salam$ ruff check 
test/tests.py:37:22: F821 Undefined name `SALAM_BIN`
   |
35 |     parent_layout_file = directory / "layout.salam"
36 |     if parent_layout_file.exists():
37 |         os.system(f"{SALAM_BIN} {parent_layout_file} > /dev/null 2>&1")
   |                      ^^^^^^^^^ F821
38 |     else:
39 |         print(
   |

Found 1 error.
```

https://github.com/astral-sh/ruff?tab=readme-ov-file#getting-started

--- a/test/tests.py
+++ b/test/tests.py
@@ -34,7 +34,7 @@ def run_tests_in_directory(directory: Path) -> None:
 
     parent_layout_file = directory / "layout.salam"
     if parent_layout_file.exists():
-        os.system(f"{SALAM_BIN} {parent_layout_file} > /dev/null 2>&1")
+        os.system(f"{salam_bin} {parent_layout_file} > /dev/null 2>&1")
     else:
         print(
             f"{COLOR_RED}{parent_layout_file} does not exist. Skipping salam command.{COLOR_RESET}"

